code-challenges.test.js

```diff
@@ -38,32 +58,135 @@ const num3 = -7
+// ● Test suite failed to run
 
+//   ReferenceError: Cannot access 'evenDivisiblebyThree' before initialization
+
+// Note to Instructor: I looked up the error and according to my research this error occurs becuase the the function set up is trying to access the variable before it's initialised so I tried restructuring my code but was getting syntax errors so I'ma bit lost because it looks correct to me. Anyway, this is why I have commented out my #1 - code lines, so I can still run tests for the other function problems.
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// describe(arrayofCapitalization, () => {
+//     it("takes in an array of words and returns an array with all the words capitalized", () => {
+//       expect(arrayofCapitalization("randomNouns1")).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//       expect(arrayofCapitalization("randomNouns2")).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+//     })
+//  })
+
+ //* RAN TEST (RESULTS BELOW): *//
+
+ // ReferenceError: arrayofCapitalization is not defined
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
-
 // b) Create the function that makes the test pass.
 
+// Pseudocode: 
+// Declare a function called arrayofCapitalization.
+// Set parameters allwordNouns
+// Use conditional to convert array into a on with all capitalization
+    // Use .join to convert to string
+    // Use .charAt(0) && .toUpperCase() to capitalize entire string
+    // Use .split to convert string back to array once capitalized
+// Return output is an array containing all capitalized words.
+
+// const arrayofCapitalization = (array) => {
+//     return array.join(randomNouns1,randomNouns2) + array.charAt(0).toUpperCase +array.split (" ")
+    
+//     }
+    
+   
+// }
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   Jest encountered an unexpected token
+
+//   Details:
+
+//   SyntaxError:
+
+// Note to Instructor: I'm at a lost here with this problem, I don't even understand why I'm getting a syntax error (on line 123) or how to correctly use/order these commands. It feels like this entire problem is in Chinese. Literally spent hours on it even though there's not much code. I really thought I was understanding Jest but I haven't even got one of these tests to come back clear and I'm not sure why. This explains why I have commented out my #2 - code lines, so I can still run tests for the other function problems.
```

Comment:
You clearly put time and effort into both of these problems.  it's the same problem as question 1.  You forgot the quotes around the function name in the describe method (line 90).  I know when I get errors that I can't explain, I will compare with a similar problem character by character to see where the issue is.  It's usually something like a typo or missed punctuation :(